modules/installation-aws-add-local-zone-locations.adoc

```diff
@@ -23,29 +23,32 @@ $ export CLUSTER_REGION="<region_name>" <1>
 ----
 <1> For `<region_name>`, specify a valid AWS region name, such as `us-east-1`.
 
-. Review the list of zones that your region contains by running the following command:
+. List the zones that are available in your region by running the following command:
 +
 [source,terminal]
 ----
-$ aws ec2 describe-availability-zones \
-    --filters Name=region-name,Values=${CLUSTER_REGION} \
-    --query 'AvailabilityZones[].ZoneName' \
+$ aws --region ${CLUSTER_REGION} ec2 describe-availability-zones \
+    --query 'AvailabilityZones[].[{ZoneName: ZoneName, GroupName: GroupName, Status: OptInStatus}]' \
+    --filters Name=zone-type,Values=local-zone \
     --all-availability-zones
 ----
 +
-Depending on the region, the list of available zones can be long. The different zones use the following naming conventions:
+Depending on the region, the list of available zones can be long. The command will return the following fields:
 +
-`${REGION}[a-z]`:: Availability zones available in the region.
-`${REGION}-LID-N[a-z]`:: Available AWS Local Zones. `${REGION}LID-N` is the zone group identifier, and `[a-z]` is the zone identifier.
-`${REGION}-wl1-LID-wlz-[1-9]`:: Available Wavelength zones.
+`ZoneName`:: The name of the Local Zone.
+`GroupName`:: The group that the zone is part of. You need to save this name to opt in.
+`Status`:: The status of the Local Zone group. If the status is `not-opted-in`, you must opt in the `GroupName` by running the commands that follow.
 
 . Export a variable to contain the name of the Local Zone to host your VPC by running the following command:
 +
 [source,terminal]
 ----
-$ export ZONE_GROUP_NAME="${CLUSTER_REGION}-<location_identifier>-<zone_identifier>" <1>
+$ export ZONE_GROUP_NAME="<value_of_GroupName>" <1>
 ----
-<1> For `<location_identifier>-<zone_identifier>`, specify the location identifier and zone identifier for the Local Zone that you selected for your region. For example, specify `nyc-1a` to use the US East (New York) Local Zone.
++
+where:
```

Comment:
Since there is a callout for this command, should we just use the `<1>` instead of "where" and explain the field there? (Otherwise there's no need for the callout since it's not being explained.)

Reply:
Hi @kcarmichael08 . Good call, as this would remain consistent with our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks). I updated it to state:

> <1> The `<value_of_GroupName>` value specifies the name of the group of the Local Zone you want to create subnets on. For example, specify `us-east-1-nyc-1` to use the zone `us-east-1-nyc-1a`, US East (New York).